ui/admin/app/controllers/scopes/scope/workers/worker/tags.js

```diff
@@ -44,6 +47,34 @@ export default class ScopesScopeWorkersWorkerTagsController extends Controller {
     );
   }
 
+  updateApiTags() {
+    const apiTags = structuredClone(this.model.api_tags);
+    const { key, value } = this.modalTag;
+    const editedValues = this.editValue.split(',').map((value) => value.trim());
```

Comment:
We can edit a tag to add more than one value at a time? Is that expected?

Reply:
From discussions with product and design, yes. This follows the same pattern we have for creating tags. I feel if we didn't support it, users might try to type `dev, qa` like they can during creation and be confused as to why they cannot here.

Reply:
Oh interesting. I had no idea you could do that in creation too, that wasn't obvious to me as a user